aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,41 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## April 11, 2022 - June 3, 2022
+
+### Communications
+
+- We're looking to improve the usability of the resources we're delivering, making the site easier to search, providing more and better examples, etc. We've got a summer intern coming on board soon to help with this.
```

Comment:
Im thinking we can reword this sentence @taysea do you think you could take a look

Reply:
I'm wondering if we leave this section out for this round and announce it when it's in progress the next time around.

We are still ideating around what these changes will be specifically, so I don't wan to prematurely announce and cause misunderstandings.

Reply:
removed